Formula/lexbor.rb

```diff
@@ -0,0 +1,116 @@
+class Lexbor < Formula
+  desc "Fast embeddable web browser engine written in C with no dependencies"
+  homepage "https://lexbor.com/"
+  url "https://github.com/lexbor/lexbor/archive/refs/tags/v1.0.0.tar.gz"
+  sha256 "e9ba3db87b077cde8ee189eb0851e8b11823c778c6844adcce537357dcca8ae1"
+  license "Apache-2.0"
+
+  depends_on "cmake" => [:build, :test]
+
+  def install
+    mkdir "build" do
+      system "cmake", "..", *std_cmake_args
+      system "make"
+      system "make", "install"
+    end
+    mkdir_p "#{prefix}/tests"
+    cp_r "test", "#{prefix}/tests/test"
+    cp_r "config.cmake", "#{prefix}/tests"
+    cp_r "version", "#{prefix}/tests"
+    (prefix/"tests/CMakeLists.txt").write cmakelists_file_content
+  end
+
+  test do
+    system "cmake", "#{prefix}/tests"
+    system "make"
+    assert_system "make test CTEST_OUTPUT_ON_FAILURE=1"
```

Comment:
Is the only way to test this to compile it into something? You can't run the browser standalone?

Reply:
I believe that these ~90 tests which I have taken from the original repo are a good way to check that everything is correct with this library 😄